File: src/애너그램만들기/Anagram.java
package 애너그램만들기;

import java.util.Scanner;

public class Anagram {
    public static int[] getAlphabetCount(String str) {
        int[] count = new int[26];
        for (int i = 0; i < str.length(); i++) {
            count[str.charAt(i) - 'a']++;
        }
        return count;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        // 공백으로 구분된 한 문자열을 받기 위해서 next()를 사용합니다.
        String a = sc.next();
        String b = sc.next();

//        int[] countA = new int[26];
//        int[] countB = new int[26];
//        for (int i = 0; i < a.length(); i++) {
//            countA[a.charAt(i) - 'a']++;
//        }
//        for (int i = 0; i < b.length(); i++) {
//            countB[b.charAt(i) - 'a']++;
//        }
        int[] countA = getAlphabetCount(a);
        int[] countB = getAlphabetCount(b);

        int ans = 0;
        for (int i = 0; i < 26; i++) {
//            if (countA[i] > countB[i]) {
//                ans += countA[i] - countB[i];
//            } else if (countB[i] > countA[i]) {
//                ans += countB[i] - countA[i];
//            }
            ans += Math.abs(countA[i] - countB[i]);
        }
        System.out.println(ans);
    }
}

File: src/애너그램만들기/Anagram.kt
package 애너그램만들기

import java.util.*

private fun getAlphabetCount(str: String): IntArray {
    val count = IntArray(26)
    for (i in 0 until str.length) {
        count[str[i] - 'a']++
    }
    return count
}

fun main() {
    val sc = Scanner(System.`in`)
    val a = sc.next()
    val b = sc.next()

    val countA = getAlphabetCount(a)
    val countB = getAlphabetCount(b)

    var ans = 0
    for (i in 0 until 26) {
        ans += Math.abs(countA[i] - countB[i])
    }
    println(ans)
}